Solutions/Threat Intelligence/Analytic Rules/DomainEntity_CommonSecurityLog.yaml

```diff
@@ -43,10 +43,10 @@ query: |
   let Domain_Indicators =
       ThreatIntelligenceIndicator
       | where TimeGenerated >= ago(ioc_lookBack)
-      | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
       | where Active == true and ExpirationDateTime > now()
```

Comment:
This where statement needs to follow the summarize statement. Same for the other KQL queries.